File: 차도희/week5.java
package sep1;

public class n12899 {
	
	class Solution {
	    public String solution(int n) {
	    	
	    	
	    	String[] num = {"4","1","2"}; // 0 1 2
	        String answer = "";
	    	
	    	while(n > 0)
	    	{
	    		int tmp = n % 3;
	    		answer = num[tmp] + answer;
	    		n /= 3;
	    		if(tmp == 0)
	    			n--;
	    	}

	        return answer;
	    }
	}
}

File: 신경준/Week5.js
function solution(n) {
    let answer = '';
    while (true) {
        if ((n % 3) === 0) {
            n = n - 1;
            let remain = 4;
            answer = remain.toString() + answer;
        }

        else {
            let remain = parseInt(n % 3);
            answer = remain.toString() + answer;
        }
        n = parseInt(n / 3);
        if (n < 3) {
            if (n != 0) {
                answer = n + answer;
            }
            break;
        }
    }

    return answer;
}
